src/components/new-request/name-check/name-check.vue

```diff
@@ -0,0 +1,809 @@
+    return this.getNameCheckErrors[NameCheckErrorType.errorStructure]
+  }
+  get hasIssuesConflictAlert (): boolean {
+    return (
+      this.conflictsExact?.length > 0 ||
+      this.hasErrorsConflictsExact ||
+      this.hasErrorsConflictsSimilar)
+  }
+  get hasIssuesConflictCaution (): boolean {
+    return this.conflictsSimilar?.length > 0
+  }
+  get hasIssuesStructureAlert (): boolean {
+    return (
+      this.hasConflictsRestricted ||
+      this.isMissingDesignation ||
+      this.hasDesignationsMismatched ||
+      this.hasDesignationsMisplaced ||
+      this.hasDesignationsCheckUse ||
+      this.hasSpecialCharacters ||
+      this.hasErrorsConflictsRestricted ||
+      this.hasErrorsdesignations ||
+      this.hasErrorsStructure)
+  }
+  get hasIssuesStructureCaution (): boolean {
+    return (
+      this.hasConflictsConditional ||
+      this.isMissingDistinctive ||
+      this.isMissingDescriptive)
+  }
+  get hasSpecialCharacters (): boolean {
+    return this.specialCharacters.length > 0
+  }
+  get infoTextTab (): string {
+    if (this.checks === 'structure-check') {
+      if (this.hasIssuesStructureAlert || this.hasIssuesStructureCaution) {
+        return 'This Name Structure Check helps you build a name according to the naming ' +
+               'rules. We recommend resolving any name structure issues for the best chance ' +
+               'of your name being approved. You can edit your name above and recheck it. ' +
+               'Read helpful tips below on how to resolve each issue.'
+      }
+      return 'This name structure check helps you build a name according to the naming rules.'
+    } else {
+      if (!this.getIsXproMras) {
+        return 'The Similar Name Check helps you find existing BC corporations that may be ' +
+               'using a similar name. The check is not exhaustive and does not guarantee name ' +
+               'availability. For the best chance of having your name approved, ensure that ' +
+               'your name is unique.'
+      }
+      return 'Extraprovincial companies must use the name in their home ' +
+             'jurisdiction, unless the exact name is already in use in BC. ' +
+             'Please note that the Similar Name Check is not exhaustive ' +
+             'and does not guarantee name availability.'
+    }
+  }
+  get loadingStructureCheck (): boolean {
+    return this.isAnalyzeDesignationPending || this.isAnalyzeStructurePending || this.isAnalyzeConflictsPending
+  }
+  get nameCheckVerdict (): string {
+    if (this.isAnalyzeConflictsPending || this.isAnalyzeDesignationPending || this.isAnalyzeStructurePending) {
+      return 'Verifying Results ...'
+    }
+    if (this.tabIconVerdict.icon === 'mdi-check-circle') {
+      return 'Ready for Review'
+    }
+    return 'Attention Required'
+  }
+  get itemsConflict (): Array<NameCheckItemIF> {
+    let items = []
+    console.log('has exact errors: ', this.hasErrorsConflictsExact)
```

Comment:
Is this log for development only? or is the plan to leave this in?

Reply:
If you need to leave this in, this probably needs a lint disable comment.

Reply:
will remove